README.md

```diff
@@ -685,6 +728,24 @@ network_connections:
       dhcp4: yes
 ```
 
+Specifying a connecting profile for an ethernet device with the `ID_PATH`:
+
+```yaml
+network_connections:
+  - name: eth0
+    type: ethernet
+    autoconnect: yes
+    # For PCI devices, the path has the form "pci-$domain:$bus:$device.$function"
+    # The profile will only match the interface at the PCI address pci-0000:00:03.0
+    match:
+      path:
+        - pci-0000:00:03.0
```

Comment:
It would be nice to also have some meaningful examples for the complex patterns.

Reply:
Added the following example:

```yaml
  - name: eth0
    type: ethernet
    # Specifying a connecting profile for an ethernet device only with the PCI address
    # pci-0000:00:01.0 or pci-0000:00:03.0
    match:
      path:
        - pci-0000:00:0[1-3].0
        - &!pci-0000:00:02.0
    ip:
      address:
        - 192.0.2.3/24
```